src/main/kotlin/org/openrewrite/RecipeMarkdownGenerator.kt

```diff
@@ -1734,6 +1735,13 @@ $cliSnippet
                 recipe.name.substring(16).replace("\\.".toRegex(), "/").lowercase(Locale.getDefault())
             } else if (recipe.name.startsWith("io.moderne")) {
                 recipe.name.substring(11).replace("\\.".toRegex(), "/").lowercase(Locale.getDefault())
+            } else if (
+                recipe.name.startsWith("ai.timefold") ||
+                recipe.name.startsWith("io.quarkus") ||
+                recipe.name.startsWith("org.apache") ||
+                recipe.name.startsWith("org.axonframework") ||
+                recipe.name.startsWith("tech.picnic")) {
+                recipe.name.replace("\\.".toRegex(), "/").lowercase(Locale.getDefault())
```

Comment:
I opted not to truncate these recipe packages, to signify that they are external to OpenRewrite/Moderne; hope that's ok!